cmd/gcsim/main.go

```diff
@@ -239,3 +276,30 @@ func openWSL(url string) error {
 	args := []string{"/c", "start", url}
 	return exec.Command(cmd, args...).Start()
 }
+
+func parseStrSeedAndWriteSample(seedStr string, outputPath string, config string, gz bool, simopt simulator.Options) error {
+	seed, err := strconv.ParseUint(seedStr, 10, 64)
+
+	if err != nil {
+		return err
+	}
+
+	return writeSample(seed, outputPath, config, gz, simopt)
+}
+
+
+func writeSample(seed uint64, outputPath string, config string, gz bool, simopt simulator.Options) error {
+	cfg, err := simulator.ReadConfig(config)
+	if err != nil {
+		return err
+	}
+
+	sample, err := simulator.GenerateSampleWithSeed(cfg, seed, simopt)
+	if err != nil {
+		return err
+	}
+	sample.Save(outputPath, gz)
+	fmt.Printf("Generated sample with seed %v to %s\n", outputPath, seed)
```

Comment:
seed and outputPath are swapped

Reply:
oops, had stdout redirected in my testing script so didn't see this. fixed.